Makefile

```diff
@@ -0,0 +1,24 @@
+
+default: bin
+
+.PHONY: all
+all:  gomod_tidy gofmt bin test
+
+.PHONY: gomod_tidy
+gomod_tidy:
+	 go mod tidy
+
+.PHONY: gofmt
+gofmt:
+	go fmt -x ./...
+
+.PHONY: bin
+bin:
+	 go build *.go
+	 hack/build.sh
```

Comment:
Do we need both these commands? Now there are two binaries, right?

Reply:
True, removed the first.